src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -137,6 +141,10 @@ public function getTypes($class, $property, array $context = [])
         $types = [];
         /** @var DocBlock\Tags\Var_|DocBlock\Tags\Return_|DocBlock\Tags\Param $tag */
         foreach ($docBlock->getTagsByName($tag) as $tag) {
+            if (is_a($tag, 'phpDocumentor\Reflection\DocBlock\Tags\InvalidTag')) {
+                return null;
```

Comment:
Shouldn't we throw here too?

Reply:
Throwing here would break applications with invalid docblocks.